src/nupic/math/ArrayAlgo.hpp

```diff
@@ -5075,7 +4976,7 @@ namespace nupic {
 
     // See comments in count_gt. We need both conditional compilation and 
     // SSE_LEVEL check.
-#if defined(NTA_OS_LINUX) || defined(NTA_OS_DARWIN)
+#if not defined(NTA_OS_WINDOWS)
```

Comment:
Actually leaving this up to you, see my comment on the PR: https://github.com/numenta/nupic.core/pull/278#issuecomment-74909035